extensions/wikia/Flags/Flags.hooks.php

```diff
@@ -1,11 +1,44 @@
 <?php
-
 namespace Flags;
 
 use Flags\Helper;
 
 class Hooks {
 
+	const FLAGS_DROPDOWN_ACTION = 'flags';
+
+	public static function onBeforePageDisplay( \OutputPage &$out, \Skin &$skin ) {
```

Comment:
This goes for the other hooks too, objects are effectively passed by reference in PHP, so you don't need the `&`.